qml/pages/ProgramDelegate.qml

```diff
@@ -26,15 +27,18 @@ ListItem {
     }
 
     Item {
-        height: parent.height - 2 * Theme.paddingSmall
+        id: listContainer
+        //height: parent.height - 2 * Theme.paddingSmall
+        height: title.height + episode.height + time.height + 2*Theme.paddingSmall
         width: parent.width - 2 * Theme.paddingMedium
         anchors.centerIn: parent
 
         Rectangle {
             id: img
             color: "black"
-            height: parent.height
+            height: parent.height - 2*Theme.paddingSmall
             width: Math.ceil((height * 16) / 9)
+            y: parent.y + Theme.paddingSmall
```

Comment:
Not needed as topMargin is already used below in anchors.

Reply:
Correct; fixed.

Reply:
That's correct, I'll fix it.